audioplayer.h

```diff
@@ -0,0 +1,34 @@
+#pragma once
+#include <Windows.h>
+#include <MMSystem.h>
+#include <iostream>
+#include <io.h>
+#include <dirent.h>
+#include <vector>
+#include <string>
+#include <algorithm>
+
+#pragma comment(lib, "winMM.lib")
+
+using namespace std;
+
+void CallMciSendString(const string& str);
+
+class AudioPlayer {
+
+ private:
+  string selectedTrackName;
+  string currentVolume = "50";
```

Comment:
It's better to store it as integer, and then cast to string in `ChangeVolume`. String takes more memory then int

Reply:
fixed 🐈